### PR TITLE
CFP/diversity/accessibility pages

### PR DESCRIPTION
**Pull request by evildmp**
- added a call for proposals page `/cfp`
- added a speaker mentors page `speaker-mentors`
- added a diversity/accessibility page `/diversity-accessibility`
- added a news section with a news article `/news/target` (I have no idea how to implement this btw)
- aded items to the menu in the base template, making a big mess of it

### CODE REVIEW

File: content/cfp.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+<p>We <strong>don’t</strong> just want to hear from the experts or famous names - other perspectives are just as valuable, so don’t let the idea that you’re not expert or famous enough stop you because <em>this won’t stop us from selecting your talk or the audience from enjoying it</em>.</p>
+
+<p>Note that there is a <a href="#financial-support">programme of financial support</a> for speakers.</p>
+
+<h3>We <em>especially</em> want proposals from...</h3>
+
+<em>People who are minorities in this field</em>
+<p><a href="/diversity-accessibility">Diversity at PyCon</a> is important to us.</p>
+
+<p>If you can help us make this event more diverse just by being there, please, do come forward with a proposal. We need you.</p>
```
Comment (Lukasa) [inline content/cfp.html]: Maybe replace "more diverse" with "more representative"? I worry that "more diverse" sounds like diversity is a checkbox item for us.

File: content/cfp.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+<em>First-time speakers</em>
+<p>If you haven’t spoken at a conference before, this is your invitation to do so. After all, we want to be remembered as the conference where you started your superstar speaking career. You’ll find that a Python/Django audience is friendly and receptive, so don’t be shy.</p>
```
Comment (Lukasa) [inline content/cfp.html]: Not sure about how I feel about "Django" being in this sentence.

File: content/cfp.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+<p>Tell us about your experiments, discoveries, solutions, achievements and other adventures, using the talk proposal form.</p>
+
+<h5>Lightning talks</h5>
+
+<p>There’ll also be opportunities for five-minute lightning talks. You don’t need to register in advance; just put your name down on the day.</p>
+
+<h4>Workshop</h4>
+<p>Can you lead a tutorial, on a Python or Django topic?</p>
+
+<p>We’ll be holding introductory workshops on our open day, and more advanced sessions alongside the sprints on Thursday and Friday.</p>
```
Comment (Lukasa) [inline content/cfp.html]: Are we doing an open day?

File: content/cfp.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+<p>Please don’t hesitate to suggest a workshop for the event, using the workshop proposal form. We’re open to all kinds of ideas, and especially ones we haven’t even thought of.</p>
+
+<h4>Code clinic</h4>
+<p>Could you be a friendly neighbourhood code doctor?</p>
+
+<p>Alongside the sprints, we’re holding code clinics, where an expert will be on hand to help with people’s Python/Django questions and problems.</p>
+
+<p>A clinic might run for an hour or two. If you like to offer a clinic on a particular subject, or even a general one, please use the workshop proposal form.</p>
```
Comment (Lukasa) [inline content/cfp.html:52]: Are we doing code clinics?

File: content/cfp.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+<h3>How to submit a proposal</h3>
+<p>There’s a proposal form for talks and a proposal form for tutorials and demonstrations. Take a look at them - they’ll make it clearer how this works.</p>
+
+<p>Don’t agonise over your proposal. Just get it in. We’ll get back to you if anything’s not clear.</p>
+
+<p>Don’t hesitate to submit more than one proposal.</p>
+
+<p>The deadline for submissions is the end of 10th May (UTC).</p>
```
Comment (Lukasa) [inline content/cfp.html]: Is it?

File: content/diversity-accessibility.html [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+<p>The Trans*Code event is an opportunity to work with others on projects and issues that particularly affect the trans community.</p>
+
+<h3>Financial assistance</h3>
+<p>We will be supporting individuals who need help to meet the costs of travel, subsistence or even taking days off work. This will include discounted or free conference tickets. <em>Speakers will automatically be eligible for this assistance if they require it.</em></p>
+
+<h4>Apply for financial assistance</h4>
+<p>We have set aside a considerable sum for financial assistance.</p>
+
+<p><em>Don't be reluctant to apply for financial assistance.</em> Many of your fellow speakers and some members of the committee will be doing the same. The money is there to help <strong>you</strong>.</p>
+
```
Comment (Lukasa) [inline content/diversity-accessibility.html:38]: I'll want to flesh this section out later. =)

File: content/diversity-accessibility.html [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+<h3>Supporter tickets</h3>
+<p>We have made <em>Diversity supporter tickets</em> available. As well as your own ticket, you can buy a ticket on behalf of an attendee who couldn’t otherwise afford it. These tickets will be allocated by an awards committee, and you or your organisation will be - unless you state otherwise - thanked publicly for your generous support.</p>
```
Comment (Lukasa) [inline content/diversity-accessibility.html]: Have we?

File: templates/base.html [hunk 8]
```diff
@@ -23,10 +23,13 @@
...
                         <li><a {% if page.title == "Venue" %} class="active"{% endif %}href="/venue/">Venue</a></li>
-                        <li><a {% if page.title == "Code Of Conduct" %} class="active"{% endif %}href="/code-of-conduct/">Conduct</a></li>
+                        <li><a {% if page.title == "Code Of Conduct" %} class="active"{% endif %}href="/code-of-conduct/">Code of Conduct</a></li>
                         <li class="halfway"><a {% if page.title == "Contact" %} class="active"{% endif %}href="/contact/">Contact</a></li>
                         <li><a {% if page.title == "Sponsorship" %} class="active"{% endif %}href="/sponsorship/">Sponsor</a></li>
                         <li><a {% if page.title == "Registration" %} class="active"{% endif %}href="/register/">Register</a></li>
+                        <li><a {% if page.title == "Call for proposals" %} class="active"{% endif %}href="/cfp/">Call for proposals</a></li>
+                        <li><a {% if page.title == "Diversity & accessibility" %} class="active"{% endif %}href="/diversity-accessibility/">Diversity & accessibility</a></li>
```
Comment (Lukasa) [inline templates/base.html]: I think HTML requires that the ampersand be encoded, but I'm never entirely sure.

Comment (ghickman) [inline templates/base.html]: If we've set the site's encoding to UTF-8 (meta tag in the `head`)then it should be unnecessary to encode this.

File: content/cfp.html (hunk 1 above)
Comment (evildmp) [inline content/cfp.html]: Was never very happy with that, it does sound a bit checkboxy.

By the way, most of this content has simply been adopted from http://2015.djangocon.eu, hebce the occasional Django that I have missed.

File: content/cfp.html (hunk 3 above)
Comment (evildmp) [inline content/cfp.html]: No, mistake.

File: content/cfp.html (hunk 4 above)
Comment (evildmp) [inline content/cfp.html:52]: I think so. We've had similar sessions before, not sure if they were called clinics exactly. Harry did one last year.

File: content/cfp.html (hunk 5 above)
Comment (evildmp) [inline content/cfp.html]: According to the timeline, yes. But to be honest it seems rather early to me.

File: content/diversity-accessibility.html (hunk 7 above)
Comment (evildmp) [inline content/diversity-accessibility.html]: I think we should!

File: content/cfp.html [hunk 9]
```diff
@@ -0,0 +1,82 @@
+type: page
+title: Call for Proposals
+slug: cfp
+body_class_hack: talks
+---
+<h2>Call for proposals</h2>
```
Comment (inglesp) [inline content/cfp.html]: This line's unnecessary, since it duplicates the page heading.

File: content/cfp.html [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+<p>This is a <strong>community conference</strong>. You’re coming to the event, and that makes <strong>you</strong> part of the community and makes this <strong>your</strong> conference. If you’ve done something that you think other people might be interested in, or that you think they should know about, <em>we would love to know about it</em>.</p>
+
+<p>We <strong>don’t</strong> just want to hear from the experts or famous names - other perspectives are just as valuable, so don’t let the idea that you’re not expert or famous enough stop you because <em>this won’t stop us from selecting your talk or the audience from enjoying it</em>.</p>
+
+<p>Note that there is a <a href="#financial-support">programme of financial support</a> for speakers.</p>
+
+<h3>We <em>especially</em> want proposals from...</h3>
+
+<em>People who are minorities in this field</em>
+<p><a href="/diversity-accessibility">Diversity at PyCon</a> is important to us.</p>
```
Comment (inglesp) [inline content/cfp.html]: We're PyCon UK!

File: content/cfp.html (hunk 1 above)
Comment (inglesp) [inline content/cfp.html]: I prefer "more representative" too.

I'm also struggling to parse this line -- I think I'm confused by "by just being there" and "come forward with a proposal".  (Plus one on the sentiment though!)

File: content/cfp.html [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+<p>Proposals on any subject are welcomed, at any level of technical expertise.</p>
+
+<p>Tell us about your experiments, discoveries, solutions, achievements and other adventures, using the talk proposal form.</p>
+
+<h5>Lightning talks</h5>
+
+<p>There’ll also be opportunities for five-minute lightning talks. You don’t need to register in advance; just put your name down on the day.</p>
+
+<h4>Workshop</h4>
+<p>Can you lead a tutorial, on a Python or related topic?</p>
```
Comment (inglesp) [inline content/cfp.html]: "On Python or a related topic"

File: content/cfp.html [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+
+<p>Please don’t hesitate to suggest a workshop for the event, using the workshop proposal form. We’re open to all kinds of ideas, and especially ones we haven’t even thought of.</p>
+
+<h4>Code clinic</h4>
+<p>Could you be a friendly neighbourhood code doctor?</p>
+
+<p>Alongside the sprints, we’re holding code clinics, where an expert will be on hand to help with people’s Python questions and problems.</p>
+
+<p>A clinic might run for an hour or two. If you like to offer a clinic on a particular subject, or even a general one, please use the workshop proposal form.</p>
+
```
Comment (inglesp) [inline content/cfp.html:53]: Could we add lunchtime events here?

File: content/cfp.html (hunk 5 above)
Comment (inglesp) [inline content/cfp.html]: I arrived at this by working backwards from the conference date.  If we give people three months to prepare a talk, and spend a month choosing talks, we get the middle of May.

Is three months more time than's required?

File: content/diversity-accessibility.html [hunk 13]
```diff
@@ -0,0 +1,92 @@
+type: page
+title: Diversity and accessibility
```
Comment (inglesp) [inline content/diversity-accessibility.html]: What do you think about talking about "inclusion" rather than "diversity"?

File: content/diversity-accessibility.html [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+slug: diversity-accessibility
+body_class_hack: talks
+---
+<h2>Diversity and accessibility at PyCon UK</h2>
```
Comment (inglesp) [inline content/diversity-accessibility.html]: Again, the heading is unnecessary.

File: content/diversity-accessibility.html [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+
+<p>Most of the global software industry is dominated by a very narrow demographic, of young, white, western men. This imbalance isn't good for anyone, and we want PyCon UK 2016 to be a milestone in the effort to address it.</p>
+
+<p>We will be making every effort to hold as inclusive an event as possible, and to shift the demographic of our audience and speakers. This is not just a vague aim, <a href="/news/target">it's a clear target</a>.</p>
```
Comment (inglesp) [inline content/diversity-accessibility.html]: This link is broken, because we don't handle links to /news/xyz yet -- I can fix this.

File: content/diversity-accessibility.html [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+
+<h3>Code of conduct</h3>
+<p>Like every PyCon, we have a prominent <a href="/codeofconduct">code of conduct</a> <strong>that will be enforced</strong>.</p>
```
Comment (inglesp) [inline content/diversity-accessibility.html]: The link should be to /code-of-conduct.

File: content/diversity-accessibility.html [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+
+<h3>Minorities in our community</h3>
+<p>It’s not simply iniquitous that the minorities of our community - for example, women, people of non-European descent, people with disabilities - find it harder to participate in it: their participation is vital to its health and needed now more than ever.</p>
+
+<p>If you belong to one of these or any other under-represented groups, we’re especially keen to have you at PyCon. We want to help redress the imbalance, make a clear statement about your value to the community affirm your position as a first-class citizen of it.</p>
```
Comment (inglesp) [inline content/diversity-accessibility.html]: "community, and affirm"

File: content/diversity-accessibility.html (hunk 7 above)
Comment (inglesp) [inline content/diversity-accessibility.html]: Agree, we should.  There's no harm in asking!

However, could we call it something other than "Diversity supporter ticket"?  Maybe just "supporter ticket"?  I think the remit of the financial aid programme is broader than encouraging diversity.

@doismellburning will have views here -- he's looking at ticket pricing now.

File: content/diversity-accessibility.html [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+
+<p>The PyCon venue is fully <strong>wheelchair-accessible</strong> and has:</p>
+
+<ul>
+<li>step-free access from the street and car-parks</li>
+<li>step-free access to all auditoriums, theatres and halls</li>
+<li>toilets suitable for wheelchair users</li>
+</ul>
+
+<p>Please contact us about any mobility assistance needs you may have during your stay in Cardiff.</p>
```
Comment (inglesp) [inline content/diversity-accessibility.html]: This could link to the contact page.

File: content/speaker-mentors.html [hunk 19]
```diff
@@ -0,0 +1,24 @@
+type: page
+title: Speaker mentor programme
+slug: speaker-mentors
+body_class_hack: talks
+---
+<h2>Speaker mentors at PyCon UK</h2>
+
+<p>For first-time speakers, non-native English speakers, unconfident or uncertain speakers or simply anyone who’d like some help with a talk - here are some people who’d like to help you.</p>
+
+<p>Speaking in front of a large audience doesn’t come easily to many people. Our conference audiences are full of <em>interesting people with interesting things to say</em> - who remain in the audience because for whatever reason, they didn’t feel confident enough to submit a talk proposal.</p>
```
Comment (inglesp) [inline content/speaker-mentors.html]: The second sentence is hard to parse on first reading.

File: templates/base.html (hunk 8 above)
Comment (inglesp) [inline templates/base.html]: Works in my browser, and we've got `<meta charset="utf-8">` in the `head`.  I think this is ok.

File: templates/base.html [hunk 20]
```diff
@@ -23,10 +23,13 @@
...
+                        <li><a {% if page.title == "Speaker mentors" %} class="active"{% endif %}href="/speaker-mentors/">Speaker mentors</a></li>
```
Comment (inglesp) [inline templates/base.html]: Can we have this in a separate pull request?  We'll need @hawkz's help to get the headings right.

I've opened #3 to discuss exactly which headings we need in the global navigation bar.

File: content/cfp.html (hunk 5 above)
Comment (evildmp) [inline content/cfp.html]: I think that a deadline for announcing speakers two months in advance is probably plenty.

File: content/cfp.html (hunk 11 above)
Comment (evildmp) [inline content/cfp.html]: No, it's correct - "on a Python topic" (which isn't the same as "on Python").

Comment (snim2) [inline content/cfp.html]: "on a Python" suggests that the workshop leader should arrive, no doubt announced by fanfare and accompanied by an entourage, riding atop a large serpent. I would like to see that too, but I suspect the venue would have some health and safety concerns.

File: templates/base.html (hunk 20 above)
Comment (inglesp) [inline templates/base.html]: @evildmp -- please could you remove this change from the pull request?  It would currently break the navigation bar.

**Comment by evildmp** [comment]
@inglesp updates added

**Comment by inglesp** [comment]
Hi @evildmp -- thanks for the updates.  Please could you remove the navbar changes from this pull request, as they'll break the page layout.
